sdk/reservations/Microsoft.Azure.Management.Reservations/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Microsoft.Azure.Management.Reservations release notes
 
+### Changes in 1.15.0-preview
```

Comment:
Update package version to preview.

Reply:
@markcowl  - Please suggest how to proceed with the API version, as the build started failing after updating the API version.
Thanks for your help.

Reply:
I have updated the version to -preview in csproj also. Please check.